azerothcore/azerothcore-wotlk
# fix(DB/creature_template): Classic Dungeon Bosses Damage Output

Increase / decrease damage output based on the official wow beastiary. 

The current damage values ​​are far too low and were only partially raised with the `DamageModifier`. These values ​​should now be adjusted to the official values ​​without `DamageModifier`.

### Team project: 
Thanks to @BarbzYHOOL and @Si1ker for the help. We discussed everything in a team and worked on it for a long time! 
Big thanks especially to @Si1ker who will fix the TBC dungeon bosses! 

It was a lot of work to filter the bosses over several days and adjust the values ​​manually!





## CHANGES PROPOSED:
-  Update `mindmg` and `maxdmg` based on the official wow beastiary in `creature_template`
-  Update `minrangedmg` and `maxrangedmg` based on the factor of increasing / decreasing `mindmg` and` maxdmg` in `creature_template`
- Update `DamageModifier` for all creatures to `1` in `creature_template`

#### In addition, the creatures that are not in the instance itself but in the entrance and are listed in Atlas Loot Enhanced have been adjusted: 

- Brainwashed Noble (596)
- Marisa du'Paige (599)
- Foreman Thistlenettle (626)
- Namdo Bizzfizzle (2683)
- Techbot (6231)

#### In addition, the creatures that are not boss at all but have been listed in Atlas Loot Enhanced have been adjusted:

- Scarlet Trainee (6575)

## ISSUES ADDRESSED:
- Closes 



## TESTS PERFORMED:
- Imported SQL script on Win 10 Pro x64
- Checked a few bosses in the game and checked the dmg output



## HOW TO TEST THE CHANGES:

**Before you import the SQL script:** Test a few bosses and note the damage output!
- Import the SQL script and check that no error occurs
- Compare the damage output from before with after

## KNOWN ISSUES AND TODO LIST:


#### Not all bosses are listed in the beastiary. I adjusted the values based on this src: https://classicdb.ch/

-  Brainwashed Noble (596)
-  Fel Steed (3864)
-  Shadow Charger (3865)
-  Kam Deepfury (1666)
-  Namdo Bizzfizzle (2683)
-  Crowd Pummeler 9-60 (6229)
-  Fallen Champion (6488)
-  Baelog (6906)
-  Galgann Firehammer (7291)
-  Dustwraith (10081)
-  Avatar of Hakkar (8443) 
-  Hurley Blackbreath (9537)
-  Jed Runewatcher (10509)
-  The Ravenian (10507)
-  Baron Rivendare (10440)
-  Black Guard Swordsmith (11121)
-  Prince Tortheldrin (11486)
-  Knot Thimblejack (14338)

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
